Examples/moltin iOS Example/Models/CustomProduct.swift

```diff
@@ -12,7 +12,7 @@ class CustomProduct: Product {
     var backgroundColor: UIColor?
 
     enum ProductCodingKeys: String, CodingKey {
-        case backgroundColor = "background_colour"
+        case backgroundColor = "backgroundColor"
```

Comment:
This is in the flow as "background_colour" (english spelling) so the string should be:
```suggestion
        case backgroundColor = "backgroundColour"
```